feat: SearchCommandProcessor, ModifyCommandProcessorByName 개발

mystje opened the pull request:
feat: SearchCommandProcessor, ModifyCommandProcessorByName 개발

- SearchCommandProcessor과 이름으로 검색하여 수정하는 ModifyCommandProcessorByName 부분 추가

mystje committed:
feat: SearchCommandProcessor 개발

각 Select type에 맞게 사용자 정보를 검색하는 Processor 개발
mock fake를 활용하여 test code 작성

mystje committed:
feat: ModifyCommandProcessor 개발, SuccessToModifyByEmployeeNumber Test code 추가

- Modify 시, 필요한 modifyType, modify employee information을 setModifyInformation()을 통해서 지정
- Modify mock, employee information을 delete -> modify -> add 으로 개발

mystje committed:
fix: EmployeeInformation 오타 수정

- EmployeeInfomation- > EmployeeInformation

mystje committed:
fix: 협의한 EmployeeInformation 구조로 변경

- struct wrapping: name, phone number

mystje committed:
eat: ModifyCommandProcessor 개발 (name modify)

- full name, first name, last name으로 검색하여 정보를 수정하는 Test case 작성

mystje committed:
Merge branch 'main' into feature/Processor

mystje committed:
Merge branch 'feature/Processor' of https://github.com/KORDH/Ace-Agent into feature/Processor

MinjiKimz commented:
Search / Modify쪽 Processor 구현 내용 확인하였습니다.
고생 많으셨습니다 :)

KORDH reviewed (approved):
Search/Modify Command Processor쪽 수정사항 확인하였습니다.
고생하셨습니다!

mystje committed:
feat: ModifyCommandProcessor mock test code (careerLevel, phone number, birthday, certi level)

- careerLevel, phone number, birthday, certi level으로 수정 하는 것을 test code 작성
- test code 충돌로 SearchTest.cpp 주석 처리

JaeseungByun commented:
Search,  Modify쪽 Processor 구현 내용 확인하였습니다.

mystje committed:
fix : employeeInformation 오타 수정
infomation -> information

mystje committed:
Merge branch 'main' into feature/Processor
